root/utility/formatUserDate.js

```diff
@@ -61,12 +61,35 @@ const patterns = {
   }],
 };
 
+const TZ_CANCUN = 'America/Cancun';
+const TZ_CHICAGO = 'America/Chicago';
+const TZ_DENVER = 'America/Denver';
+const TZ_LISBON = 'Europe/Lisbon';
+const TZ_LOSANGELES = 'America/Los_Angeles';
+const TZ_NEWYORK = 'America/New_York';
+const TZ_PARIS = 'Europe/Paris';
+const TZ_PHOENIX = 'America/Phoenix';
+const TZ_SOFIA = 'Europe/Sofia';
+
+const timeZoneFallbacks = {
+  CET: TZ_PARIS,
+  CST6CDT: TZ_CHICAGO,
+  EET: TZ_SOFIA,
+  EST: TZ_CANCUN,
+  EST5EDT: TZ_NEWYORK,
+  MET: TZ_PARIS,
+  MST: TZ_PHOENIX,
+  MST7MDT: TZ_DENVER,
+  PST8PDT: TZ_LOSANGELES,
+  WET: TZ_LISBON,
+};
```

Comment:
That would break next daylight saving time change when user’s computers switch back to CEST or WEST, see this more complete list of [time zone abbreviations](https://www.timeanddate.com/time/zones/) (which contains homonyms).

Another option would be to use `jstimezonedetect` which doesn’t return abbreviations. (Yes I know it is yet another dependency, but it is much lighter than `moment` and it just _gives you the zone info key representing your device's time zone setting_, no less no more.)

Reply:
I'm a bit confused about what you mean. The names in `timeZoneFallbacks` aren't being guessed or detected at all, they're what the user selected in /account/preferences. There's no option for e.g. CEST in the list; Perl's `DateTime` correctly switched to CEST where needed with CET set for the time zone.

Reply:
Sorry, I thought that name was guessed from user’s browser:
https://github.com/metabrainz/musicbrainz-server/blob/db2845bb84eb6399395657800975ff2fbc7ea548/root/static/scripts/account/components/PreferencesForm.js#L111
but it is indeed filtered with:
https://github.com/metabrainz/musicbrainz-server/blob/db2845bb84eb6399395657800975ff2fbc7ea548/root/static/scripts/account/components/PreferencesForm.js#L115-L124
which doesn’t allow for other abbreviations.